common.yml

```diff
@@ -660,6 +661,24 @@ tasks:
       - func: "upload release packages to s3"
       - func: "upload release packages to linux repos"
 
+  - name: check-snyk-results
+    tags: ["git_tag"]
+    depends_on:
+      - name: push
+        variant: "*"
+    commands:
+      - func: "fetch source"
+      - command: expansions.update
+      - command: shell.exec
+        params:
+          working_dir: src/github.com/mongodb/mongo-tools
+          script: |
+            ${_set_shell_env}
+            set -e
+            curl -o ./snyk https://static.snyk.io/cli/latest/snyk-linux
```

Comment:
I think in the long term we should consider store this file internally.

Reply:
Do you mean store it in the repo? This isn't FOSS, so I wouldn't want to put it in this repo.

Reply:
I am not thinking about storing it in a repo. Maybe a permanent S3 storage? Maybe we can ask the DevProd team for idea.

Reply:
We already download all sorts of other stuff from the internet at large. What makes this particular thing different?

Reply:
This is an external dependency which is not managed by any packaging system either. Let's say if synk website is down for one day, our release will be blocked.

Reply:
This is an external dependency which is not managed by any packaging system either. Let's say if synk website is down for one day, our release will be blocked.

Reply:
Again, we already have a number of other similar unmanaged deps (precious, etc.). If there were a well organized way of vendoring these things that wasn't just a one off that'd be great. I will bring this up with DevProd.

Reply:
Yea, that's why I said we should do it "in the long term". We don't need to do it now. Besides, I feel more confident to download from Github than other third party websites, since if GitHub goes down, we probably won't be able to release anyway.

Reply:
Fair point re: GitHub. FWIW, `static.snyk.io` appears to be served via Akamai, so I'd expect it to be fairly reliable.